openmc/material.py

```diff
@@ -310,6 +313,47 @@ def from_hdf5(cls, group: h5py.Group):
 
         return material
 
+    @classmethod
+    def from_NCrystal(cls, cfg):
+        """Create material from NCrystal configuration string
+        Density is set from the NCrystal value,
+        and material temperature from the configuration string.
+      
+        Parameters
+        ----------
+        cfg : str
+            NCrystal configuration string
+
+        Returns
+        -------
+        openmc.Material
+            Material instance
+
+        """
+
+        try:
+            import NCrystal
+        except ImportError:
+            raise SystemExit("ERROR: NCrystal Python module is required.")
+
+        NC_mat = NCrystal.createInfo(cfg)
+        NC_comp = NC_mat.getComposition()
```

Comment:
On the Python side, we also use lower-case with underscores for local variables
```suggestion
        nc_mat = NCrystal.createInfo(cfg)
        nc_comp = NC_mat.getComposition()
```